.github/workflows/ci.yml

```diff
@@ -41,6 +41,8 @@ jobs:
 
     - uses: conda-incubator/setup-miniconda@v2
       with:
+        mamba-version: "*"
+        channels: conda-forge,defaults
```

Comment:
Can we avoid adding defaults as a channel here and elsewhere, if possible? Having multiple channels tends to slow things down a bit.

Reply:
Yep, works.